complex/ft_main.c

```diff
@@ -161,6 +180,9 @@ static void ft_fw_convert_info(struct fi_info *info, struct ft_info *test_info)
 	info->ep_attr->type = test_info->ep_type;
 	info->ep_attr->protocol = test_info->protocol;
 	info->ep_attr->protocol_version = test_info->protocol_version;
+	
+	info->tx_attr->msg_order = FI_ORDER_SAW;
+	info->rx_attr->msg_order = FI_ORDER_SAW;
```

Comment:
We shouldn't set these unless the ordering is actually needed -- we're testing RMA writes.